core/sodasql/cli/ingest.py

```diff
@@ -232,13 +342,35 @@ def ingest(
     if not soda_server_client.api_key_id or not soda_server_client.api_key_secret:
         raise ValueError("Missing Soda cloud api key id and/or secret.")
 
-    if tool == 'dbt':
-        dbt_manifest, dbt_run_results = resolve_artifacts_paths(
-            dbt_artifacts=dbt_artifacts,
-            dbt_manifest=dbt_manifest,
-            dbt_run_results=dbt_run_results
-        )
-        test_results_iterator = map_dbt_test_results_iterator(dbt_manifest, dbt_run_results)
+    if tool == "dbt":
+        if (
+            dbt_artifacts is not None
```

Comment:
Shouldn't this be 
```
 dbt_artifacts is not None
 or (dbt_manifest is not None and dbt_run_results is not None)
```
?

Reply:
nope, that is handled in the `load_dbt_artifacts` (previously the `resolve_dbt_artifacts`)

Reply:
Ok - I'd expect the validation to happen in `download_dbt_artifacts_from_cloud` function instead of here though.

Reply:
done